cle/backends/__init__.py

```diff
@@ -497,10 +528,14 @@ def set_arch(self, arch):
         self.memory = Clemory(arch) # Private virtual address space, without relocations
 
     @property
+    def image_base_delta(self):
+        return - self.linked_base + self.mapped_base
```

Comment:
I can replace with just `self.mapped_base - self.linked_base`. I saw something like that in the codebase, just thought it's more clear.